radixconfig_playground.yaml

```diff
@@ -43,8 +48,8 @@ spec:
           monitoring: true
           resources:
             requests:
-              memory: "500Mi"
-              cpu: "200m"
+              memory: "8192Mi"
+              cpu: "2000m"
```

Comment:
Why do we request mpre more memory and CPU than x86 on playground, but not on production? Is this due to the (large) tests?

Reply:
These now are the same configs as in production/test-large.

As I was comparing performance I wanted specs to be as similar as possible now. Decided to include in the commit as it makes more sense to have expected characteristics whenever we need to recreate playground environment.